dist/components/ui/msm-modal/msm-modal.scss

```diff
@@ -0,0 +1,236 @@
+@media (min-width: 768px) {
+  .modal-mobile {
+    display: none;
+  }
+
+  .modal-backdrop {
+    &.in {
+      opacity: 0.65;
+    }
+  }
+
+  .modal {
+    text-align: center;
+    padding: 0 !important;
+  }
+
+  .modal:before {
+    content: '';
+    display: inline-block;
+    height: 100%;
+    vertical-align: middle;
+    margin-right: -4px;
+  }
+
+  .modal-dialog {
+    display: inline-block;
+    margin: 0;
+    text-align: left;
+    vertical-align: middle;
+  }
+
+  .modal-content {
+    -webkit-box-shadow: none;
+    -moz-box-shadow: none;
```

Comment:
No vendor prefixes, please. Use `autoprefixer`, if you need special prefixes in your project. https://github.com/postcss/autoprefixer